src/compounds/billboard/src/stories.tsx

```diff
@@ -118,6 +118,11 @@ const fullWidthSlot = (
   </Row>
 )
 
+const bgImage: string = text(
```

Comment:
why dont we read a local static file?

Reply:
We can also use the static file. With this, we can change the source url on storybook without changing the code and see how different images look like. @erkinun